### ISSUE TITLE
[Feature] Filter the Reviews view by Program (OMSCS/OMSA)

### ISSUE THREAD

**Issue by romcabrera** [martzcodes/OMSCentral#27]

**Comment by martzcodes** [comment]
I guess it would make sense to add a section for OMSCS vs OMSA... it is OMS Central after all...

This will require a bit more rework

**Comment by romcabrera** [comment]
What about just adding a field to specify for each class whether there are OMSCS or OMSA.
Then at the Courses view it could be possible to provide a toggle "Show omscs/omsa/all courses"

**Comment by ynvtlmr** [comment]
Potentially related:
[Feature] Filter view by specialization

**Comment by martzcodes** [comment]
@romcabrera yeah that's what i was thinking.  I'll just back-classify all previous reviews as OMSCS and give the option to change it to OMSA moving forward
